### PR TITLE
Uwshievmod 51: email user and research librarians package publish results

### PR DESCRIPTION
**Pull request by Tao-Liu**

### CODE REVIEW

File: app/mailers/notifier.rb [hunk 1]
```diff
@@ -72,4 +72,22 @@ def notify_user_reset_password_instructions(recipient)
           :subject => "#{system_name} - Reset password instructions")
   end
 
+  def notify_recipients_of_successful_package_publish(package, recipients)
+    @system_name = system_name
+    @package = package
+    mail(:to =>recipients,
+        :from => APP_CONFIG['notification_email_sender'],
+        :reply_to => APP_CONFIG['notification_email_sender'],
+        :subject => "#{system_name} - Package publishing is success")
```
Comment (dyazbek-intersect) [inline app/mailers/notifier.rb]: "Package publishing is successful"

File: app/mailers/notifier.rb [hunk 2]
```diff
@@ -72,4 +72,22 @@ def notify_user_reset_password_instructions(recipient)
...
+  end
+
+  def notify_recipients_of_failed_package_publish(package, recipients)
+    @system_name = system_name
+    @package = package
+    mail(:to => recipients,
+         :from => APP_CONFIG['notification_email_sender'],
+         :reply_to => APP_CONFIG['notification_email_sender'],
+         :subject => "#{system_name} - Package publishing is failed")
```
Comment (dyazbek-intersect) [inline app/mailers/notifier.rb]: Package publishing has failed

File: app/views/notifier/notify_recipients_of_failed_package_publish.html.haml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+%p Hello,
+%p Unable to publish package #{@package.filename}.
```
Comment (dyazbek-intersect) [inline app/views/notifier/notify_recipients_of_failed_package_publish.html.haml]: The package #{@package.filename} could not be published.

File: app/views/notifier/notify_recipients_of_successful_package_publish.html.haml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+%p Hello,
+%p Package #{@package.filename} has been successfully submitted for publishing.
```
Comment (dyazbek-intersect) [inline app/views/notifier/notify_recipients_of_successful_package_publish.html.haml]: The package #{@package.filename} has been published successfully.

File: db/migrate/20150903010750_add_research_librarians_to_system_configuration.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+class AddResearchLibrariansToSystemConfiguration < ActiveRecord::Migration
+  def change
+    add_column :system_configurations, :research_librarians, :string, default: ""
```
Comment (dyazbek-intersect) [inline db/migrate/20150903010750_add_research_librarians_to_system_configuration.rb]: You don't need a migration for each field you add... You can put both fields in one migration.

**Comment by dyazbek-intersect** [comment]
:+1:

**Comment by Tao-Liu** [comment]
Comments addressed and merged

**Pull request merged**